src/renderer/components/pages/MaintenancePage.tsx

```
@@ -157,9 +199,64 @@ export const MaintenancePage: React.FC = () => {
+
+  const fRunFilter = useCallback(async () => {
+    if (!fDest || !hasLibrary || fValidRules.length === 0) return;
+    const opId = `filter-${Date.now()}`;
+    fOpIdRef.current = opId; fCancelledRef.current = false;
+    setFPhase('running'); setFProgress(null); setFResult(null); setFError(null);
+    try {
+      const res = await window.electronAPI.filterLibrary({
+        operationId: opId, tracks, libraryPath: libraryPath ?? '',
+        filters: fValidRules, options: { destination: fDest, mode: fMode, conflictResolution: fConflict },
+      });
+      if (fCancelledRef.current) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)